scripts/LedSignMock.py

```diff
@@ -0,0 +1,37 @@
+import curses
+import argparse
+
+
+
+class LedSignMock(data):
+    parser = argparse.ArgumentParser(description="Prints mock output.")
```

Comment:
the parser should be in the rpc server. just put it at the top with all the other imports

so we can do this:
```
# show it
python server/led_sign/led_sign_server.py --mock-output
# dont show it (default)
python server/led_sign/led_sign_server.py
```